src/advanced_agent/interfaces/fastapi_gateway.py

```
@@ -184,6 +184,131 @@ async def chat_completions(
+                raise
+            except Exception as e:
+                logger.error(f"セッション取得エラー: {e}")
+                raise HTTPException(status_code=500, detail=str(e))
+        
+        # システム統計エンドポイント
+        @self.app.post(f"/{API_VERSION}/system/stats")
+        async def system_stats(request: dict, _: bool = Depends(self._verify_api_key)):
+            """システム統計"""
+            try:
+                include_gpu = request.get("include_gpu", True)
+                include_memory = request.get("include_memory", True)
```

Ruff (0.12.2)

Local variable `include_memory` is assigned to but never used

Remove assignment to unused variable `include_memory`

(F841)